R/profile_plot.R

```diff
@@ -130,24 +131,31 @@ profile_plot <- function(mydir, rep, para, profilesummary) {
 
   dev.off()
 
-  maxyr <- min(profilesummary$endyrs)
+  maxyr <- min(profilesummary$endyrs + 1)
```

Comment:
I just want to make sure that we are not doing anything with estimates of `F` right? If we were then, for some values we would want endyr + 1 and for F we would want endyr.

Reply:
In this function no. The maxyr here is only applied here to determine the spawing biomass/output and fraction unfished in the first projection year.

Reply:
Thanks for confirming @chantelwetzel-noaa.